### PR TITLE
Add pow charts

### PR DESCRIPTION
**Pull request by Jujhar**
Addresses #90 

Brings back PoW chart.

powchart0

The other charts make sense 
pow-screen

but the main one doesn't should I remove the _All_ selector?

### CODE REVIEW

**Comment by Sirmorrison** [comment]
Are you plotting the chart for all the pow sources?

**Comment by Jujhar** [comment]
Yeah that was there when I set it up but if you go and pick a pool then it looks good with it displaying hashrate so I added it with a workers option too.

**Comment by Jujhar** [comment]
All PoW charts makes sense now

Screenshot from 2019-07-26 15-41-59
Screenshot from 2019-07-26 15-55-00
Screenshot from 2019-07-26 15-53-53
Screenshot from 2019-07-26 15-54-17

**Comment by Jujhar** [comment]
Added data label to x axis, removed scientific notation on y axis, and removed size field. 

Screenshot from 2019-07-27 08-30-46

**Comment by raedah** [comment]
There needs to be dates showing on the X axis.

**Comment by Sirmorrison** [comment]
> 
> 
> Added data label to x axis, removed scientific notation on y axis, and removed size field.
> 
> ![Screenshot from 2019-07-27 08-30-46](https://user-images.githubusercontent.com/5521110/61996459-6aa7e500-b049-11e9-9dab-dca3c680e44c

File: web/public/app/src/controllers/pow_controller.js [hunk 1]
```diff
@@ -127,28 +152,133 @@ export default class extends Controller {
...
+      if (_this.dataType === 'workers') {
+        dataTypeLabel = 'Workers'
+      }
+
+      var extra = {
+        labels: ['Date', 'luxor', 'uupool', 'btc', 'f2pool', 'coinmine'],
+        colors: ['#2971FF', '#FF8C00', '#64FFDA', '#84FFFF', '#EEFF41', '#FFCCBC'],
+        labelsDiv: this.labelsTarget,
+        ylabel: dataTypeLabel,
+        y2label: 'Network Difficulty',
```
Comment (Sirmorrison) [inline web/public/app/src/controllers/pow_controller.js]: Add `digitsAfterDecimal: 8` to the extra var to fix the issue with the scientific notation.

Comment (Jujhar) [reply web/public/app/src/controllers/pow_controller.js]: digitsAfterDecimal does not fix it, still there if this is added, but this is already fixed, I used  using labelsKMB.

**Pull request closed without merging**